### PR TITLE
Add Cloudflare API docs changelog

### PR DESCRIPTION
**Pull request by jhutchings1**
### Summary

Adds a changelog for the new API docs site. 

### Documentation checklist

- [x] The [documentation style guide](https://developers.cloudflare.com/style-guide/) has been adhered to.
- [ ] If a larger change - such as adding a new page- an issue has been opened in relation to any incorrect or out of date information that this PR fixes.
- [ ] Files which have changed name or location have been allocated [redirects](https://developers.cloudflare.com/pages/configuration/redirects/#per-file).

### CODE REVIEW

File: src/content/changelogs/fundamentals.yaml [hunk 1]
```diff
@@ -5,6 +5,12 @@ productLink: "/fundamentals/"
 productArea: Core platform
 productAreaLink: /fundamentals/reference/changelog/platform/
 entries:
+  - publish_date: "2024-12-16"
+    title: Cloudflare API docs are now automatically generated
+    description: | - 
+      Cloudflare's API documentation is now being automatically generated based on OpenAPI Schemas, and we have retired our old documentation. The move to OpenAPI Schemas allows us to ensure greater consistency and quality across our API documentation. The documentation now also includes examples of how to call the API using curl, Typescript, Python, and Go.
```
Comment (jacobbednarz) [inline src/content/changelogs/fundamentals.yaml]: non-blocking: perhaps a more generic term for our SDKs to make this more applicable for longer?

```suggestion
      Cloudflare's API documentation is now being automatically generated based on OpenAPI Schemas, and we have retired our old documentation. The move to OpenAPI Schemas allows us to ensure greater consistency and quality across our API documentation. The documentation now also includes examples of how to call the API using curl and our SDKs. 
```

**Review by jacobbednarz — APPROVED** [review]

File: src/content/changelogs/fundamentals.yaml (hunk 1 above)
Comment (jhutchings1) [reply src/content/changelogs/fundamentals.yaml]: ~Sure, but I'm going to Oxford that comma back in 😉~ 

Ok, changed my mind. It's better without the comma for a list of two

File: src/content/changelogs/fundamentals.yaml [hunk 2]
```diff
@@ -5,6 +5,12 @@ productLink: "/fundamentals/"
 productArea: Core platform
 productAreaLink: /fundamentals/reference/changelog/platform/
 entries:
+  - publish_date: "2024-12-16"
+    title: Cloudflare API docs are now automatically generated
+    description: | - 
+      Cloudflare's API documentation is now being automatically generated based on OpenAPI Schemas, and we have retired our old documentation. The move to OpenAPI Schemas allows us to ensure greater consistency and quality across our API documentation. The documentation now also includes examples of how to call the API using curl or our SDKs. 
+
+      Refer to [Cloudflare API documentation](/api), or the [blog post on our transition to OpenAPI](https://blog.cloudflare.com/open-api-transition/) for more information.
```
Comment (patriciasantaana) [inline src/content/changelogs/fundamentals.yaml]:
```suggestion
    description: |- 
      Cloudflare's API documentation is now being automatically generated based on OpenAPI Schemas, and we have retired our old documentation. The move to OpenAPI Schemas allows us to ensure greater consistency and quality across our API documentation. The documentation now also includes examples of how to call the API using curl or our SDKs. 

      Refer to the [Cloudflare API documentation](/api), or the [blog post on our transition to OpenAPI](https://blog.cloudflare.com/open-api-transition/) for more information. 
```

File: src/content/changelogs/fundamentals.yaml [hunk 3]
```diff
@@ -5,6 +5,12 @@ productLink: "/fundamentals/"
 productArea: Core platform
 productAreaLink: /fundamentals/reference/changelog/platform/
 entries:
+  - publish_date: "2024-12-16"
+    title: Cloudflare API docs are now automatically generated
+    description: |- 
+      Cloudflare's API documentation is now being automatically generated based on OpenAPI Schemas, and we have retired our old documentation. The move to OpenAPI Schemas allows us to ensure greater consistency and quality across our API documentation. The documentation now also includes examples of how to call the API using curl or our SDKs. 
+
+      Refer to the [Cloudflare API documentation](/api), or the [blog post on our transition to OpenAPI](https://blog.cloudflare.com/open-api-transition/) for more information.
```
Comment (patriciasantaana) [inline src/content/changelogs/fundamentals.yaml]:
```suggestion
    description: |- 
      Cloudflare's API documentation is now being automatically generated based on OpenAPI Schemas, and we have retired our old documentation. The move to OpenAPI Schemas allows us to ensure greater consistency and quality across our API documentation. The documentation now also includes examples of how to call the API using curl or our SDKs. 

      Refer to the [Cloudflare API documentation](/api/), or the [blog post on our transition to OpenAPI](https://blog.cloudflare.com/open-api-transition/) for more information. 
```

**Review by patriciasantaana — APPROVED** [review]

**Pull request merged**